sapi/fpm/tests/tester.inc

```diff
@@ -1253,6 +1253,37 @@ class Tester
         return true;
     }
 
+    public function expectLogConfigOptions(array $options)
+    {
+        $configOptions = $this->getConfigOptions();
+        foreach ($options as $name => $value) {
+            if (!isset($configOptions[$name])) {
+                return $this->error("Expected config option: {$name} = {$value} but {$name} is not set");
+            }
+            if ($configOptions[$name] != $value) {
```

Comment:
```suggestion
            if ($configOptions[$name] !== $value) {
```

Reply:
This one tricky. I will not check type because parse from string line that means values always be string type.

Reply:
Ok that makes sense